tests/ut/ir/transforms/test_collect_comm_groups.py

```
@@ -329,6 +378,32 @@ def host_orch(self):
+    distinct branch from ``test_dead_alloc_no_dispatch_raises`` (which has a
+    view but no consuming dispatch).
+    """
+
+    @pl.program
+    class P:
+        @pl.function(level=pl.Level.HOST, role=pl.Role.Orchestrator)
+        def host_orch(self):
+            buf = pld.alloc_window_buffer(1024)  # noqa: F841  # never windowed
+            return 0
+
+    with pytest.raises(Exception, match="no pld.tensor.window materialisation"):
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)